book/projects/project_roadmap.md

```diff
@@ -13,25 +13,12 @@ Each project will progress through these stages:
 
 ### Weeks Before the Event
 
-Ideally the project work begins in advance of the hackweek. As soon as you are accepted to the event we will invite you to Slack where you can begin sharing project ideas:
+We will form project teams in the month prior to the hackweek. As soon as you are accepted to the event we will invite you to Slack where you can begin exploring project ideas. Organizers will be available to answer questions and to help participants get connected with a project that fits their learning objectives.
 
-{fa}`check,text-success mr-1` Have an idea for a project? Pitch it on the hackweek Slack group #project-ideas channel
-
-{fa}`check,text-success mr-1` If you don't yet have an idea brewing, take time to browse other ideas on Slack. Start up a conversation in a Slack thread if something catches your eye.
-
-{fa}`check,text-success mr-1` Feel free to consult with one of the organizers if you have questions about project purpose, scope, roles and datasets.
-
-{fa}`check,text-success mr-1` You may already have a critical mass forming. This is great! Go ahead and begin defining [project roles](project_roles) so that you can arrive at the hackweek ready to get started.
 
 ### Day 1 of the Event
 
-We facilitate a project team formation session at the end of the first day of the event.
-
-{fa}`check,text-success mr-1` People with project ideas will make a 1-minute pitch to the group
-
-{fa}`check,text-success mr-1` People looking for a project are invited into small groups to learn more about existing ideas
-
-{fa}`check,text-success mr-1` UW eScience helpers and facilitators are available to help teams get clear on size and scope of the effort
+We facilitate a project team formation session on the first day of the hackweek. During this time we will finalize project goals and objectives. Hackweek organizers and facilitators will help each team set reasonable goals and develop a roadmap for task completion.

```

Comment:
A link to the project repo template could be useful here (note that the link in project_initialization is broken)